src/mixins/view.js

```diff
@@ -221,9 +222,15 @@ const ViewMixin = {
       }
       parent = parent._parent;
     }
+  },
+
+
+  // Overwrite https://github.com/jashkenas/backbone/blob/b8c054aec729cdc1d93e55fbe3e9f23016ed4da7/backbone.js#L1267
+  _removeElement() {
```

Comment:
actually this _could_ just replace line `143` now?  Also if we want to keep this function for some reason can we move it up with `destroy`?